modules/ROOT/pages/administration/access-control/load-privileges.adoc

```diff
@@ -0,0 +1,82 @@
+:description: How to use Cypher to manage load privileges.
+
+////
+[source, cypher, role=test-setup]
+----
+CREATE ROLE roleLoadAllData
+----
+////
+
+[role=enterprise-edition aura-db-enterprise]
+[[access-control-load-privileges]]
+= Load privileges
+
+[abstract]
+--
+This section explains how to use Cypher to manage load privileges.
+--
+
+All load privileges are relevant system-wide.
+Like DBMS privileges, they do not belong to one specific database or graph.
+For more details on the differences between graphs, databases and the DBMS, refer to xref::introduction/cypher_neo4j.adoc[].
+
+image::privileges_grant_and_deny_syntax_load_privileges.svg[title="Syntax of GRANT and DENY load Privileges"]
+
+// TODO: add image later when there is more than one LOAD privilege
+//image::privileges_hierarchy_load.svg[title="Load privileges hierarchy"]
+
+The load privileges applies to the xref:clauses/load-csv.adoc[`LOAD CSV` clause], filtering whether the data can be loaded from the given source.
+
+The load privileges are assignable using Cypher administrative commands.
+They can be granted, denied and revoked like other privileges.
+
+.Load privileges command syntax
+[options="header", width="100%", cols="3a,2"]
+|===
+| Command | Description
+
+| [source, syntax, role=noheader]
+GRANT [IMMUTABLE] LOAD
+  ON ALL DATA
+  TO role[, ...]
+| Enables the specified roles to load external data in queries.
+
+|===
+
+[NOTE]
+====
+More details about the syntax descriptions can be found xref:administration/index.adoc#administration-syntax[here].
+====
+
+Unlike other privileges these are not granted, denied, or revoked on `DBMS`, `Database`, or `Graph` but instead on the data source to load from.
+Currently, only on `ALL DATA` meaning all sources.
```

Comment:
Are there concrete plans to change it? Otherwise, I would avoid "currently".
```suggestion
Adding `ON ALL DATA` means a role has the privilege to load data from all sources.
```

Reply:
Do we want to point out that it is (currently) the only alternative? or we skip that completely?

Reply:
I think we skip. The docs tend to not make reference to future changes to the product.